packages/rps/src/redux/open-games/saga.ts

```diff
@@ -1,16 +1,26 @@
-import { fork, take, select, cancel, call, apply } from 'redux-saga/effects';
```

Comment:
Not really specific to this PR but I think a line width of 80 is a bit too small 😛